unfoldingWord/translationCore
# Updated (ulb, udb, translationWords) to latest resources

#### This pull request addresses:
- Updated (ulb, udb, translationWords) to latest resources.
- Added a migration action that will ensure we delete the user's resources folder every time a new tC version is released.

#### How to test this pull request:
- Temporary change the app version in the package.json (This will trigger the resources migration code that deletes your local resources folder and regenerates it again for you)
- Verify you see version 10 for ulb and udb and version 6 for translationWords.
- basically, test both tools and multiple projects and make sure it renders all resources.
- try loading all bibles available in the scripture pane.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2387)
